specification/commerce/resource-manager/Microsoft.Commerce/preview/2015-06-01-preview/commerce.json

```diff
@@ -88,6 +98,17 @@
               "$ref": "#/definitions/UsageAggregationListResult"
             }
           },
+          "202": {
+            "description": "Response indicating that a request has been accepted for processing. However, the processing has not been completed.",
```

Comment:
Hi @pilor , thanks for your review, Joe raised this pr to fixed an S360 item which complains 202 is not defined. As we discussed offline, although 202 is not allowed in GET apis, but this behavior was released 2014. https://learn.microsoft.com/en-us/previous-versions/azure/reference/mt219001(v=azure.100)?redirectedfrom=MSDN. Could you kindly help to approve this change?

Reply:
please remove the long running operation attribute

Reply:
I removed the long running operation attribute.

Just for background: I originally added this because the linter told me that this was required for the 202. I think removing this will generate another alert.

Reply:
@weidongxu-microsoft , this api is not a LRO but the service takes several hours to aggregate the data to returned, 202 is returned since the api was released in year2014

Reply:
Even we allow 202, why it is ErrorResponse?

HTTP client will timeout in a few minutes. If API takes hours but not an LRO, it just means it won't return anything useful.

Fine with it. Java likely will just refuse release anything unless this solved.